### PR TITLE
Manual Scoop of HoC LevelBuilder content

### PR DESCRIPTION
**Pull request by aaronwaggener**
For triage on 11/30
Manual scoop of 5 files required for Hour of Code. 4 are for Poetry and one is a foorm survey.

### CODE REVIEW

**Review by mikeharv — APPROVED** [review]
All changes to poetry files are expected and look good to me!

File: dashboard/config/foorm/forms/surveys/pd/eir_baseline_practices_survey_admin.6.json [hunk 1]
```diff
@@ -0,0 +1,1624 @@
+{
```
Comment (maddiedierker) [inline dashboard/config/foorm/forms/surveys/pd/eir_baseline_practices_survey_admin.6.json:1]: did someone request that we ship this survey? i'm not too concerned about it (it seems low risk), mostly just wondering

File: dashboard/config/scripts/levels/poetry_hoc_exemplar2.level [hunk 2]
```diff
@@ -40,14 +40,17 @@
       "poem art predict_library"
     ],
     "preload_asset_list": null,
+    "encrypted_examples": [
+
+    ],
```
Comment (maddiedierker) [inline dashboard/config/scripts/levels/poetry_hoc_exemplar2.level:45]: @code-dot-org/curriculum-platform it looks innocuous, but is an empty array for `encrypted_examples` expected here? i would've assumed that the value would either be `null` or an encrypted string (like the change to `poetry_hoc_spriteSizes.level` below), but i'm not super familiar with this so i want to double-check

Comment (davidsbailey) [reply dashboard/config/scripts/levels/poetry_hoc_exemplar2.level:45]: good question Maddie! it's a bit strange that this diff is showing up, especially given that this level has been edited recently. overall this looks like a cosmetic change to me and I wouldn't be worried about it breaking anything, though.

**Review by davidsbailey — APPROVED** [review]
The changes to the .script_json files look good to me. I'm not too familiar with the other content. I would just note that we've been having some problems with PDF generation lately, so it could be worth checking that the PDFs for these scripts/lessons are working on staging before we DTP this change. e.g. https://staging-studio.code.org/s/poetry-2021?viewAs=Instructor --> Printing Options

**Review by maddiedierker — APPROVED** [review]

File: dashboard/config/foorm/forms/surveys/pd/eir_baseline_practices_survey_admin.6.json (hunk 1 above)
Comment (aaronwaggener) [reply dashboard/config/foorm/forms/surveys/pd/eir_baseline_practices_survey_admin.6.json:1]: Yes, request came in from Tess and Baker

**Pull request merged**